### ISSUE TITLE
AltServer Installation Failed

### ISSUE THREAD

**Issue by israeljr10** [rileytestut/AltStore#251]
**AltServer to iPhone Installation Failed**
When I click "install AltStore" then selected my iPhone and follow the iCloud procedures for installation, an error will pop-up
saying INSTALLATION FAILED create directory: The system cannot find the path specified.:  "C:\Users\İsrael\AppData\Local\Temp\91f1d363-0467-49c9-b7b9-f10f87608b4f
But these numbers in the error message changes whenever I retry to install again to iPhone

 - OS: [Windows 10 (intel)]
![Screenshot (50)](https://user-images.githubusercontent.com/66537977/83956609-9ea68600-a868-11ea-886e-3b8a7b66d864.png)

**Comment by StellaMeow** [comment]
Have you solved this issue?

**Comment by israeljr10** [comment]
> Have you solved this issue?

No, I have not yet solved it. Do you have any idea on how I can solve it, please?

**Comment by StellaMeow** [comment]
> > Have you solved this issue?
> 
> No, I have not yet solved it. Do you have any idea on how I can solve it, please?

Unfortunately no. I’m having this issue as well and have tried some advice got from Reddit but none worked.

**Comment by lidortoz** [comment]
> > > Have you solved this issue?
> > 
> > 
> > No, I have not yet solved it. Do you have any idea on how I can solve it, please?
> 
> Unfortunately no. I’m having this issue as well and have tried some advice got from Reddit but none worked.

i got the same issue. please someone tell me he fixed it .

**Comment by StellaMeow** [comment]
Hi guys, I found my issue lay in the foreign characters used in my account name. I created a new local account using only English characters and installed AltServer in this new account again, it worked.

**Comment by lidortoz** [comment]
> Hi guys, I found my issue lay in the foreign characters used in my account name. I created a new local account using only English characters and installed AltServer in this new account again, it worked.

You mean I need to create a new Apple ID and use it instead of my present one?

**Comment by StellaMeow** [comment]
> > Hi guys, I found my issue lay in the foreign characters used in my account name. I created a new local account using only English characters and installed AltServer in this new account again, it worked.
> 
> You mean I need to create a new Apple ID and use it instead of my present one?

No, I mean you need to create a new Windows local account.

**Comment by lidortoz** [comment]
> > > Hi guys, I found my issue lay in the foreign characters used in my account name. I created a new local account using only English characters and installed AltServer in this new account again, it worked.
> 
> > 
> 
> > You mean I need to create a new Apple ID and use it instead of my present one?
> 
> 
> 
> No, I mean you need to create a new Windows local account.

And can I have both new account and old account with no data loss? Sorry for askin but I don't understand that much about windows 10...

**Comment by StellaMeow** [comment]
> > > > Hi guys, I found my issue lay in the foreign characters used in my account name. I created a new local account using only English characters and installed AltServer in this new account again, it worked.
> > 
> > 
> > > 
> > 
> > 
> > > You mean I need to create a new Apple ID and use it instead of my present one?
> > 
> > 
> > No, I mean you need to create a new Windows local account.
> 
> And can I have both new account and old account with no data loss? Sorry for askin but I don't understand that much about windows 10...

You won’t lose any data in the old account and you can switch between these two accounts easily.

**Comment by lidortoz** [comment]
> > > > > Hi guys, I found my issue lay in the foreign characters used in my account name. I created a new local account using only English characters and installed AltServer in this new account again, it worked.
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > You mean I need to create a new Apple ID and use it instead of my present one?
> 
> > > 
> 
> > > 
> 
> > > No, I mean you need to create a new Windows local account.
> 
> > 
> 
> > And can I have both new account and old account with no data loss? Sorry for askin but I don't understand that much about windows 10...
> 
> 
> 
> You won’t lose any data in the old account and you can switch between these two accounts easily.

Thank you , I will try that tomorrow🙂
